tests/element_test_vectors.rs

```diff
@@ -50,10 +50,6 @@ trait RoundTrip {
 /// integration tests to dictate which readers/writers should be used in each test.
 struct NativeElementWriterApi;
 
-// TODO: This implementation is not yet being used; the native reader still paired with the ion-c
-//       writer in the interest of minimizing the number of places that test failures need to be
-//       researched. Once the native reader is passing enough ion-tests, we'll enable the
-//       native writer so we can fix any failures it produces.
```

Comment:
Looks like there's another outdated `TODO` below about handling `TextKind::Pretty` if you feel like removing it while you're here.

Reply:
I saw this, but it didn't seem to make sense with the surrounding the code, so I left it in to deal with later. Are you saying that there is no code to clean up for this, and I can just remove the comment?

Reply:
I think it's just an outdated comment that can be removed.

Reply:
So I went to remove the comment you mentioned... and _somehow_ I accidentally added support in the tests for `TextKind::Lines`... and replaced all of the repetitive "good round trip" tests with a macro. And I fixed up a doc test that referenced ion-c.

Now I'm at `+68 -4262` for this PR... and there are an additional 3456 tests covering round trips to and from the `Lines` format.